lib/zendesk_api/resource_ticket.rb

```diff
@@ -0,0 +1,116 @@
+module ZendeskAPI
+  # https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/
+  class Ticket < Resource
```

Comment:
Do we need tests for these new classes?

Reply:
this is an extraction, the tests are already in place.